### PR TITLE
`amp-date-countdown` component to support remaining-time in milliseconds

### PR DESCRIPTION
**Pull request by leonalicious**
I would like to add  `timeleft-ms` option in order to support for the remaining-time in milliseconds requirement.

\cc @aghassemi @cathyxz @cvializ

### CODE REVIEW

**Comment by leonalicious** [comment]
@cvializ done with the protoascii file, and I have also set the `git config user.email` with my CLA email on the last commit, so, will the CLA takes effect soon or I do need to re-create this PR and submit with my CLA commit instead ?

File: extensions/amp-date-countdown/validator-amp-date-countdown.protoascii [hunk 1]
```diff
@@ -38,7 +38,7 @@ tags: {  # <amp-date-countdown>
   }
   attrs: {
     name: "end-date"
-    mandatory_oneof: "['end-date', 'timestamp-ms', 'timestamp-seconds']"
+    mandatory_oneof: "['end-date', 'timestamp-ms', 'timestamp-seconds','timeleft-ms']"
```
Comment (honeybadgerdontcare) [inline extensions/amp-date-countdown/validator-amp-date-countdown.protoascii]: nit: please alphabetize, `timeleft-ms` should be after `end-date`. same with others below

File: extensions/amp-date-countdown/validator-amp-date-countdown.protoascii [hunk 2]
```diff
@@ -66,14 +66,19 @@ tags: {  # <amp-date-countdown>
...
     value_regex: "\\d{13}"
   }
   attrs: {
     name: "timestamp-seconds"
-    mandatory_oneof: "['end-date', 'timestamp-ms', 'timestamp-seconds']"
+    mandatory_oneof: "['end-date', 'timestamp-ms', 'timestamp-seconds','timeleft-ms']"
     value_regex: "\\d{10}"
   }
+  attrs: {
```
Comment (honeybadgerdontcare) [inline extensions/amp-date-countdown/validator-amp-date-countdown.protoascii]: nit: we try to keep attributes in order alphabetically by name, so this should be higher up.

**Comment by leonalicious** [comment]
@cvializ I have done the rebase with my CLA email address but seems like it is not working as expected, any clue please ?

**Comment by cvializ** [comment]
If you look at the list of commits, there are still some with the previous email address. It looks like you created a merge commit, which kept the commits with the previous email address. Maybe you can move your changes to a new branch and make a new PR? Or if you're comfortable with squashing your commits into one and force-pushing to this branch that could also be an option.

**Comment by leonalicious** [comment]
Will close this PR and please refer https://github.com/ampproject/amphtml/pull/18415

**Pull request closed without merging**